### ISSUE TITLE
MainLoop.NOTIFICATION_WM_QUIT_REQUEST notification is not being handled in BSPWM

### ISSUE THREAD

**Issue by ishidex2** [godotengine/godot#35490]
**Godot version:** 3.1.2 and higher

**OS/device including version:** Linux mint 19.3 / BSPWM

**Issue description:**
When window is being closed in bspwm, the MainLoop.NOTIFICATION_WM_QUIT_REQUEST notification is not being handled

**Steps to reproduce:**
use bspwm and put this script anywhere in scene
```
func _notification(what):
	if what == MainLoop.NOTIFICATION_WM_QUIT_REQUEST:
		print("Gracefully exiting")
		get_tree().quit()
```
**Minimal reproduction project:** 
reproduce steps above

**Comment by akien-mga** [comment]
You mention `NOTIFICATION_WM_QUIT_REQUEST` but your code handles `NOTIFICATION_WM_UNFOCUS_REQUEST`.

**Comment by ishidex2** [comment]
Sorry, typo, still doesn't resolve the problem

**Comment by xdee** [comment]
Did you add

`get_tree().set_auto_accept_quit(false)`

in _ready function.

**Comment by ishidex2** [comment]
yes

**Comment by akien-mga** [comment]
Should be fixed by #37247.
